project/Dependencies.scala

```diff
@@ -12,7 +12,7 @@ object Dependencies {
 
   val workbenchUtilV    = "0.5-4c7acd5"
   val workbenchModelV   = "0.11-2bddd5b"
-  val workbenchGoogleV  = "0.18-6942040"
+  val workbenchGoogleV  = "0.18-39dba9f-SNAP"
```

Comment:
Is there an associated wb-libs PR? We try not to have `-SNAP` dependencies.

Reply:
i'll have to check with matt. i think there's something somewhere

Reply:
ta-da https://github.com/broadinstitute/workbench-libs/pull/213